modules/ismapper/ismapper.nf

```diff
@@ -0,0 +1,37 @@
+process ISMAPPER {
+
+    input:
+        tuple val(sampleName), val(meta), path(reads)
+        path(ref_gbk)
+        path(ref_fasta)
+        path(query_multifasta)
+
+
+    output:
+        tuple val(meta), path("ismapper/*"), emit: results
+
+    script:
+
+        """
+
+        ismap \\
+            --t $task.cpus \\
+            --output_dir $sampleName \\
+            --queries $query_multifasta \\
+            --reference $ref_gbk \\
+            --reads $reads
+
+        # Reorganize output files
+        mkdir ismapper
+        mv $sampleName/*/* ismapper/
+
+
+        # FIXME This would need to updated later for accommodating various static lengths per insertion element. Work with a CSV file name,element_sequence,sequence_length
+        convert_ismapper_to_vcf.py \\
+            --is_mapper_file \\
+            --reference_file $ref_fasta \\
+            --te_file \\
+            --output_vcf_file output.vcf
+
+        """
```

Comment:
This part is still failing, due to the absence of `transposable_elements` in the ISMAPPER output.

Reply:
Also, not sure if `ismapper` would even produce this field 

See https://github.com/jhawkey/IS_mapper?tab=readme-ov-file#single-isolate-output

Reply:
This will be resolved by the Python script itself.